src/Extensions/src/App.Metrics.Extensions.HealthChecks/App.Metrics.Extensions.HealthChecks.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="App.Metrics.Core" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions" Version="2.2.0" />
```

Comment:
Looks like target too

Reply:
If you mean reference to `Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions` .. it does not exist in 2.1 - we have to reference 2.2 here.